client/src/components/sources/sourceConstants.js

```diff
@@ -16,3 +16,23 @@ export const SourceFilterFields = [
     ]
   }
 ];
+
+const SourceSortFields = [
```

Comment:
Note: We should probably come up with some casing guidelines for bits other than classes. I've been uppercasing (yelling) my constants, may be able to add that to Prettier/Eslint

Reply:
I don't mind uppercase for simple constants. Not sure I'd want to see it here.